docs/userGuide/contentAuthoring.md

```diff
@@ -576,4 +576,12 @@ Note:
 - Footers should not be nested in other components or HTML tags. If found inside, they will be shifted outside to be rendered properly.
 - [MarkBind components](#use-components) and [includes](#include-contents) are not supported in footers.
 
-</div>
+### Using Anchors
+
+Sometimes, you may want a user of your site to be able to obtain a link to a heading within a page, to share with someone else for example.

```

Comment:
_Nit:_
~~Sometimes, you may want a user of your site~~ **Your site's viewers may want** to ~~be able to~~ obtain a link to a heading within a page, to share with someone else for example.

Reply:
We don't use the word "user", there's a part in the documentation that uses the word "viewer", so let's use that to be more consistent.

<sub>Side note: though I think "visitor" actually sounds more accurate, but can't change it now :P.</sub>

Reply:
Or "reader" as in [<kbd>a-ReaderUsability</kbd>](
https://github.com/MarkBind/markbind/labels/a-ReaderUsability) label.